### ISSUE TITLE
Any plan to support also Server?

### ISSUE THREAD

**Issue by malcommac** [pusher/NWWebSocket#22]
Hi guys,
Thank you for your work. Is there any plan to support also websocket server?

**Comment by danielrbrowne** [comment]
Are you referring to exposing `NWWebSocketSever` via the public API? 

If so, do you have a use case in mind? In its current implementation the server isn't really suitable for use outside of the unit tests as it has too many baked-in assumptions IMHO. However, in the future a server object could be made available via the public API if there is a demand for it.

**Comment by malcommac** [comment]
Yeah. Currently there are several implementation of websocket client for iOS, few using SwiftNIO.  
However there is only one implementation (Telegraph from the far I know) which support even the server side.  
SwiftNIO allows to create both server and client and I think it's a nice addition to the library supports both of them. I'm working on a software which needs a server on macOS and an iOS client and currently there aren't so many options to use.
Thanks.
